export.py

```diff
@@ -1779,6 +1779,21 @@ def has_emissive_material(db):
     return False
 
 
+def export_for_bake(db):
+    export = True
+    node_names = []
+    for mat in db.material:
+        if mat and mat.node_tree:
+            nt = mat.node_tree
+            for n in nt.nodes:
+                node_names.append(n.name)
+    if node_names:
+        if 'PxrBakeTexture' not in node_names:
+            export = False
+
```

Comment:
for mat in db.materials:
	if mat and mat.node_tree:
		for n in mat.node_tree.nodes:
			if n.bl_idname == “PxrBakeTexturePatternNode”
				return True
		return False
	else:
		return True